levelkdev/swapr-dapp
# New user dashboard

# Summary

Fixes #1288 
<img width="720" alt="Screenshot 2022-08-26 at 16 53 38" src="https://user-images.githubusercontent.com/102727631/186933016-c1831e97-48cc-46f9-8ae1-09431c4a4d3a.png">
<img width="286" alt="Screenshot 2022-08-26 at 16 53 50" src="https://user-images.githubusercontent.com/102727631/186933049-888f77bd-5216-4ff3-9e64-044c6a622fb7.png">



  # To Test
Go to user dashboard from the ENS name or your wallet address. You should be able to see your recent transactions

  # Background
